### PR TITLE
add cypress test case for simple poll flow

### PR DESCRIPTION
**Pull request by kerryarchibald**
1.
- create an open poll
- vote, update vote
- display bot votes and bot vote changes

2.
- create an open poll
- reply to it in a thread
- bot votes in poll, check votes are displayed and updated in thread and main timeline

(https://github.com/vector-im/element-web/issues/22413)

## Checklist

* [ ] Tests written for new code (and old code if feasible)
* [ ] Linter and other CI checks pass
* [ ] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: cypress/support/composer.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+}
+
+Cypress.Commands.add("getComposer", (isRightPanel?: boolean): Chainable<JQuery> => {
+    const panelClass = isRightPanel ? '.mx_RightPanel' : '.mx_RoomView_body';
+    return cy.get(`${panelClass} .mx_MessageComposer`);
+});
+
+Cypress.Commands.add("openMessageComposerOptions", (isRightPanel?: boolean): Chainable<JQuery> => {
+    return cy.getComposer(isRightPanel).within(() => {
```
Comment (weeman1337) [inline cypress/support/composer.ts:39]: I may be wrong but [as far as I understand Cypress](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress#Commands-Run-Serially) we can probably use this simpler code here:

```
cy.getComposer(isRightPanel).within(() => {
    cy.get('[aria-label="More options"]').click();
});
return cy.get('.mx_MessageComposer_Menu');
```

File: cypress/support/composer.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+/// <reference types="cypress" />
+
+import Chainable = Cypress.Chainable;
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
+            // Scroll to the top of the timeline
```
Comment (weeman1337) [inline cypress/support/composer.ts:25]: Can you check the comments? They don't fit the commands.

**Review by weeman1337 — CHANGES_REQUESTED** [review]
Thanks for the test :slightly_smiling_face:

Found some minor issues. I'm curious to see if the code simplification works.

File: cypress/support/composer.ts (hunk 1 above)
Comment (kerryarchibald) [reply cypress/support/composer.ts]: Looks good! I find the kind-of-async-then-chaining part of cypress a bit confusing

Comment (weeman1337) [reply cypress/support/composer.ts]: +1 from me. Takes some time getting used to it

**Review by weeman1337 — APPROVED** [review]

**Pull request merged**